docs/quickstart.md

```diff
@@ -40,72 +42,278 @@
-➜  ~ dce accounts list
-[{"accountStatus":"NotReady","adminRoleArn":"arn:aws:iam::555555555555:role/MasterAcctAcces","createdOn":1572562180,"id":"555555555555","lastModifiedOn":1572637591,"principalPolicyHash":"\"852ee9abbf1220a111c435a8c0e65490\"","principalRoleArn":"arn:aws:iam::555555555555:role/DCEPrincipal-dcelogin"}]
+1. Type `dce accounts list` to verify that your account has been added.
 
+    ```
+    ➜  ~ dce accounts list
+    [
+        {
+            "accountStatus": "NotReady",
+            "adminRoleArn": "arn:aws:iam::555555555555:role/DCEMasterAccess",
+            "createdOn": 1575485630,
+            "id": "775788068104",
+            "lastModifiedOn": 1575485630,
+            "principalPolicyHash": "\"bc5872b50475b186afea67ff47516a8f\"",
+            "principalRoleArn": "arn:aws:iam::775788768154:role/DCEPrincipal-quickstart"
+        }
+    ]
+    ```
     The account status will initially say `NotReady`. It may take up to 5 minutes for the new account to be processed. Once the account status is `Ready`, you may proceed with creating a lease.
 
-10. Now that your accounts pool isn't emtpy, you can create your first lease using the `dce leases create` command
+## Leasing a DCE Account
+
+1. Use the `dce auth` command to authenticate as a non-admin user.
+
+1. Now that your accounts pool isn't empty, you can create your first lease using the `dce leases create` command.
+
+    ```
+    ➜  ~ dce leases create --budget-amount 100.0 --budget-currency USD --email jane.doe@email.com --principal-id jdoe99
+    Lease created: {
+        "accountId": "555555555555",
+        "budgetAmount": 100,
+        "budgetCurrency": "USD",
+        "budgetNotificationEmails": [
+            "jane.doe@email.com"
+        ],
+        "createdOn": 1575490207,
+        "expiresOn": 1576095007,
+        "id": "e501cb86-8317-458b-bdce-d47ab92f86a8",
+        "lastModifiedOn": 1575490207,
+        "leaseStatus": "Active",
+        "leaseStatusModifiedOn": 1575490207,
+        "leaseStatusReason": "Active",
+        "principalId": "jdoe99"
+    }
+    ```
+   
+1. Type `dce leases list` to verify that a lease has been created
+
+    ```
+    ➜  ~ dce leases list
+   [
+   	{
+   		"accountId": "555555555555",
+   		"budgetAmount": 100,
+   		"budgetCurrency": "USD",
+   		"budgetNotificationEmails": [
+   			"jane.doe@email.com"
+   		],
+   		"createdOn": 1575490207,
+   		"expiresOn": 1576095007,
+   		"id": "e501cb86-8317-458b-bdce-d47ab92f86a8",
+   		"lastModifiedOn": 1575490207,
+   		"leaseStatus": "Active",
+   		"leaseStatusModifiedOn": 1575490207,
+   		"leaseStatusReason": "Active",
+   		"principalId": "jdoe99"
+   	}
+   ]
+    ```
+
+1. Notice that [we created a lease for a different user](https://github.com/Optum/dce/issues/137) than the one we are currently authenticated with (i.e. `jdoe99` != `quickstartuser`). If we try to login to this leased account, we will receive a permissions error since it is registered under a different user.
```

Comment:
> we created a lease for a different user

this is actually a bug in our system, that a user can create a lease for someone else.
I would avoid highlighting this bug in our docs.